src/styles/code-read.css

```
@@ -396,8 +782,8 @@
   appearance: none;
   -webkit-appearance: none;
 
-  padding: calc(14px * var(--pr-scale)) calc(40px * var(--pr-scale))
-    calc(14px * var(--pr-scale)) calc(18px * var(--pr-scale));
+  padding: calc(10px * var(--pr-scale)) calc(40px * var(--pr-scale))
+    calc(10px * var(--pr-scale)) calc(18px * var(--pr-scale));
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)